internal/collector/gte_pg16_metrics.yaml

```diff
@@ -0,0 +1,126 @@
+# This list of queries configures an OTel SQL Query Receiver to read pgMonitor
+# metrics from Postgres.
+#
+# https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/-/receiver/sqlqueryreceiver#metrics-queries
+# https://github.com/CrunchyData/pgmonitor/blob/development/sql_exporter/common/crunchy_global_collector.yml
+
+# NOTE: Some of the columns below can return NULL values, for which sqlqueryreceiver will warn.
```

Comment:
Link to https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/sqlqueryreceiver#null-values ?

Reply:
Are we using COALESCE below to return 0 and avoid NULL?

Reply:
Yep.

Reply:
I've updated the comment with the link and mentioned how COALESCE was used.